### DISCUSSION TITLE
科技爱好者周刊（第 311 期）：低利率与长期项目 | ruanyf-weekly

### DISCUSSION THREAD

**Discussion by giscus** [plantree/press-comment#40] [General]
# 科技爱好者周刊（第 311 期）：低利率与长期项目 | ruanyf-weekly

阮一峰的技术周刊

https://ruanyf-weekly.vercel.app/weekly/issue-311

**Comment by suredream** [comment]
`DuckDB as the New jq` 的link错了，链到 duckdb上了，应该是：

https://www.pgrs.net/2024/03/21/duckdb-as-the-new-jq/

**Reply by plantree** [reply]
感谢！已经更新。
